docs/application/vscode-ext/web.md

```diff
@@ -79,232 +52,236 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
 
-  1. Open **Command Palette** and enter **Run Tizen App**.
-  2. Enter the folder name of the project you want to run. (Folder name, not path)
-  3. When the run is complete, it succeeds, and there is successful generation of launch app popup. Also, you will see a build success log in the terminal at the bottom of the vscode.
+To run project:
 
+  1. In VS Code, click **View** > **Command Palette**.
+  2. In the input field of the **Command Palette** that appears, type **Run Tizen App** and press **Enter**.
+  3. Enter the folder name of the project that you want to run. 
+      > **Note**
+      >
+      > Ensure that you enter the name of the folder and not the path.
+  4. When the installation is successfully completed, a install successful notification appears. Also, you will see a build success log in the terminal at the bottom of the vscode.
   ![Run Your Tizen App](media/tizen_web15.png)
 
-> **Note**  
->
-> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
->
-> In addition, vscode terminal displays successful launch log messages:
-> 
-> 1. debug 0 means normal launch.
-> 2. debug 1 means launching an Inspector available.
+        > **Note**  
+        >
+        > If you have a Tizen device and want to deploy the application into it, then you must first generate a certificate and add it.
+        >
+        > In addition, `vscode` terminal displays successful launch log messages:
+        >
+        > - debug `0` means normal launch.
+        > - debug `1` means launching an inspector available from command line.
 
 ### Debug Application in Emulator
 
-To debug application, follow these steps:
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Debug Tizen App** and press **Enter**.
+3. Enter the name of the Tizen Web App you want to run. 
 
-> **Note**
->
-> Ensure the following:
-> - Emulator is launched
-> - Device is connected to the PC
-> - Chrome browser is installed
+    A successful notification appears and the app is launched. Also, you will see a successful launch log in the terminal on the vscode.
 
-1. Open **Command Palette** and enter **Debug Tizen App**.
-2. Enter the name of the Tizen Web App you want to run.
-3. A successful popup is created and the app is launched.
+    - debug `0` means normal launch.
+    - debug `1` means launching an inspector available from command line.
 
-> **Note**
-> 
-> In addition, `vscode` terminal displays successful launch log messages:
->
-> - debug `0` means normal launch.
-> - debug `1` means launching an inspector available from command line
-> - additionally, debug the connected port as shown in the following log
+    When port information is displayed in **Terminal** to **OUTPUT**, it means that it is launched in debug mode.
```

Comment:
Line 211: Please change **Terminal** to **OUTPUT** to:

**OUTPUT**

Reply:
done.